src/webui/abstractwebapplication.cpp

```diff
@@ -395,18 +394,12 @@ QString AbstractWebApplication::saveTmpFile(const QByteArray &data)
     return QString();
 }
 
-QStringMap AbstractWebApplication::initializeContentTypeByExtMap()
+const QStringMap AbstractWebApplication::CONTENT_TYPE_BY_EXT =
 {
-    QStringMap map;
-
-    map["htm"] = Http::CONTENT_TYPE_HTML;
-    map["html"] = Http::CONTENT_TYPE_HTML;
-    map["css"] = Http::CONTENT_TYPE_CSS;
-    map["gif"] = Http::CONTENT_TYPE_GIF;
-    map["png"] = Http::CONTENT_TYPE_PNG;
-    map["js"] = Http::CONTENT_TYPE_JS;
-
-    return map;
-}
-
-const QStringMap AbstractWebApplication::CONTENT_TYPE_BY_EXT = AbstractWebApplication::initializeContentTypeByExtMap();
+    std::make_pair("htm", Http::CONTENT_TYPE_HTML),
```

Comment:
Using this constructor: http://doc.qt.io/qt-5/qmap.html#QMap-1
Any suggestion to make it more concise?